compose.yml

```
@@ -0,0 +1,8 @@
+services:
+  preguntonic-backend:
+    container_name: preguntonic-backend
+    build:
+      context: .
+      dockerfile: Dockerfile
+    ports:
+      - 8080:8080
```

If you are not able to open the project in localhost add this:
```suggestion
      - 127.0.0.1:8080:8080
```